atspi-common/src/events/object.rs

```diff
@@ -416,12 +416,33 @@ pub struct StateChangedEvent {
 	pub item: crate::events::ObjectRef,
 	/// The state to be enabled/disabled.
 	pub state: State,
-	/// Enabled or disabled the state.
-	///
-	/// 1 == enabled
-	///
-	/// 0 == disabled
-	pub enabled: i32,
+	/// Whether the state was enabled or disabled.
+	#[serde(with = "i32_bool_conversion")]
```

Comment:
Interesting! I had no idea this way of custom deserialization of a field existed.